lib/internal/source_map/source_map_cache.js

```diff
@@ -351,11 +352,53 @@ function findSourceMap(sourceURL) {
   return sourceMap;
 }
 
+/**
+ *
+ * @param {Record<string, unknown>} callSite // The call site object to reconstruct from source map
+ * @returns {Record<string, unknown> | undefined} // The reconstructed call site object
+ */
```

Comment:
These are validated so they are not really unknown. 


```suggestion
/**
 * @typedef {object} CallSite
 * @property {string} scriptName
 * @property {string} functionName
 * @property {number} lineNumber
 * @property {number} columnNumber
 */

/**
 * @param {CallSite} callSite // The call site object to reconstruct from source map
 * @returns {CallSite | undefined} // The reconstructed call site object
 */
```